tests/components/vallox/test_config_flow.py

```diff
@@ -0,0 +1,254 @@
+"""Test the Vallox integration config flow."""
+from unittest.mock import patch
+
+from vallox_websocket_api.exceptions import ValloxApiException
+
+from homeassistant.components.vallox.config_flow import host_valid
```

Comment:
Don't import this here.  Instead start a flow with `await hass.config_entries.flow.async_init` , pass invalid data, and verify the result

Reply:
Good point 👍 Turns out I already had tests for invalid hosts through `hass.config_entries.flow.async_init`, so I just removed the two tests that used this import.